cmd/provision/main.go

```diff
@@ -62,8 +62,9 @@ const (
 	// appletDataNumBlocks is the number of blocks in the applet data storage area.
 	appletDataNumBlocks = 0x400000
 
-	// Flag template for provisioning CI devices
-	template_ci = "ci"
+	// Flag template values for the different standard environments
+	template_ci   = "ci"
+	template_prod = "prod"
```

Comment:
Should we call this `wave0`, wdyt?

Reply:
There's a whole conversation here :-)

I figured these might be moving targets, i.e. "ci" changes to the new log when we rotate, and maybe we create "ci-archive1" or something. That way --template=prod or --template=ci always gets you onto the latest train. Other option is to treat like these git version tags and give them versioned names, e.g. prod0 (or wave0)

Reply:
Renamed to wave0.